### PR TITLE
Add namespace for distro mapping for SLES

### PR DESCRIPTION
**Pull request by dspalmer99**
This PR updates grype-db to support feed data from the SLES namespace. I'm marking this as a draft PR until that data is available in the feed service and I can test with it.

This Pr makes the following changes:
- Bumps the syft version to get the SLES distro
- Adds namespace for distro mapping for SLES
- Updates the unit test

See [https://github.com/anchore/grype/issues/315](https://github.com/anchore/grype/issues/315) for more context on this change. I don't believe there's a corresponding grype-db issue, but I am happy to create one if it is helpful.

Signed-off-by: Dan Palmer <dan.palmer@anchore.com>

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -6,7 +6,7 @@ require (
 	github.com/alicebob/sqlittle v1.4.0
 	github.com/anchore/go-version v1.2.2-0.20200810141238-330bef18dbca
 	github.com/anchore/grype v0.14.1-0.20210702143224-05ade7bbbf70
-	github.com/anchore/syft v0.19.0
+	github.com/anchore/syft v0.22.0
```
Comment (kzantow) [inline go.mod:9]: Where did this `0.22.0` come from, just prepping for the next release?

Comment (dspalmer99) [reply go.mod:9]: [https://github.com/anchore/syft/releases/tag/v0.22.0](https://github.com/anchore/syft/releases/tag/v0.22.0) Which I see now has not been released. I bumped it to this version specifically to get the addition of the SLES distro type [here](https://github.com/anchore/syft/blob/2f99a35f51dde8c84443ce038b71a748439cf83b/syft/distro/type.go#L20) which, from manual testing, doesn't look like it was in v0.21.0.

**Review by wagoodman — APPROVED** [review]

**Review by zhill — APPROVED** [review]
lgtm

**Comment by dspalmer99** [comment]
Thanks @wagoodman and @zhill

**Pull request merged**